### PR TITLE
Strimzi 0.33 update

### PR DESCRIPTION
**Pull request by matzew**
## Proposed Changes

- Updating just the Strimzi Operator to 0.33
- Putting in the _pure_ manifest from upstream, and doing `sed` to replace namespaces... since https://github.com/knative-sandbox/eventing-kafka-broker/pull/2381 we do have some "patched" (hard-coded) `namespace: kafka` references on the manifest, which this PR changes

0.33 release notes: https://github.com/strimzi/strimzi-kafka-operator/releases/tag/0.33.0
Some highlights:
* Use Java 17 as the runtime for all containers...
* Improved FIPS (Federal Information Processing Standards) support

### CODE REVIEW

**Review by aliok — APPROVED** [review]
/lgtm
/approve

I am fine with it if tests are fine too :)

**Comment by haubenr** [comment]
@dilipgb Dilip, can you please do the review for this PR? Thanks!

**Comment by aliok** [comment]
/hold

Holding for @dilipgb 's review

**Comment by dilipgb** [comment]
/lgtm
/approve
/remove-hold

It looks fine to me. I'm good to merge if tests are passed. Removing the hold.

**Pull request merged**
